### PR TITLE
Add link for interactive tools

### PR DESCRIPTION
**Pull request by anuprulez**

### CODE REVIEW

**Comment by bgruening** [comment]
Thanks @anuprulez!

**Comment by jmchilton** [comment]
This confused Anton also, I think this is a usability bug that can be backported to 19.09 even. Thanks so much.

**Review by jmchilton — APPROVED** [review]

File: client/galaxy/scripts/components/ToolEntryPoints/ToolEntryPoints.vue [hunk 1]
```diff
@@ -25,12 +25,15 @@
             </ul>
         </div>
 
-        You may also access all active InteractiveTools from the User menu.
+        You may also access all active InteractiveTools from the
+        <a :href="appRoot() + 'interactivetool_entry_points/list'">User menu</a>.
```
Comment (dannon) [inline client/galaxy/scripts/components/ToolEntryPoints/ToolEntryPoints.vue]: This should probably be a computed property.

**Comment by dannon** [comment]
TS test is unrelated to this.  Thanks for the fix, and the tweaks @anuprulez

**Pull request merged**